integration-tests/jgit/src/test/java/io/quarkus/it/jgit/JGitTest.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.it.jgit;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.core.Is.is;
+
+import org.arquillian.smart.testing.rules.git.server.EmbeddedHttpGitServer;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class JGitTest {
+
+    private final EmbeddedHttpGitServer gitServer = EmbeddedHttpGitServer
+            .fromBundle("booster-catalog", "repos/booster-catalog.bundle").usingAnyFreePort().create();
+
+    @BeforeEach
+    void startGitServer() throws Exception {
+        gitServer.start();
+    }
+
+    @Test
+    void should_clone() {
```

Comment:
Shouldn't camel case be used everywhere?

Reply:
Could you please also squash to a single commit?

Reply:
I could, but isn't the big green Squash button enough? :smile:

Reply:
Not the way it is used in Quarkus 😎. In Quarkus a merge commit is always used, so committers are usually asked to squash there commits.
GitHub doesn't have a feature where you could squash commits and create a merge-commit as well unfortunately.

Reply:
Thanks!